extensions-contrib/opencensus-extensions/src/test/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParserTest.java

```diff
@@ -124,6 +125,32 @@ public void testGaugeParse() throws Exception
     Assert.assertEquals(2000, row.getMetric("value").doubleValue(), 0.0);
   }
 
+  @Test
+  public void testIntGaugeParse() throws Exception
+  {
+    //configure parser with desc file
+    OpenCensusProtobufInputRowParser parser = new OpenCensusProtobufInputRowParser(parseSpec, null, null, "");
+
+    DateTime dateTime = new DateTime(2019, 07, 12, 9, 30, ISOChronology.getInstanceUTC());
+
+    Timestamp timestamp = Timestamp.newBuilder().setSeconds(dateTime.getMillis() / 1000)
+        .setNanos((int) ((dateTime.getMillis() % 1000) * 1000000)).build();
+
+    System.out.println(timestamp.getSeconds() * 1000);
```

Comment:
please remove any `System.out` statements from test code before committing. If you need to output anything use a logger. In this case, I don't think there is much value in these outputs.

Reply:
let's also remove that statement from `testDoubleGaugeParse` above.

Reply:
Yeah, just didn't notice the statement in both methods.